### PR TITLE
Integrate notification api via ember-model-table on notification-logs

### LINKED ISSUE #623: Implement notification api on admin reports system logs
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend
image

### PR DESCRIPTION
**Pull request by CosmicCoder96**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:

Integrate notification api via ember-model-table on notification-logs

Fixes #623

### CODE REVIEW

**Comment by niranjan94** [comment]
Deployment link: https://pr-628-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-628-open-event-frontend-fossasia.surge.sh

File: app/controllers/admin/reports/system-logs/notification-logs.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import Ember from 'ember';
+
+const { Controller } = Ember;
+
+export default Controller.extend({
+  columns: [
+    {
+      propertyName : 'user',
```
Comment (geekyd) [inline app/controllers/admin/reports/system-logs/notification-logs.js]: We must use the correct propertyName here. For displaying the user's name we must have `user.first-name` here.

File: app/controllers/admin/reports/system-logs/notification-logs.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      template     : 'components/ui-table/cell/admin/reports/system-logs/notification-logs/cell-time',
+      title        : 'For'
+    },
+    {
+      propertyName : 'receivedAt',
```
Comment (geekyd) [inline app/controllers/admin/reports/system-logs/notification-logs.js]: Use snake case here.

**Comment by niranjan94** [comment]
Deployment link: https://pr-628-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-628-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-628-open-event-frontend-fossasia.surge.sh

File: app/components/ui-table-server.js [hunk 3]
```diff
@@ -89,7 +89,7 @@ export default ModelsTable.extend({
     let query = extend({}, get(data, 'query'));
     let store = get(data, 'store');
     let modelName = get(data, 'type.modelName');
-    query.filter = JSON.parse(query.filter);
+    query.filter = JSON.parse(query.filter || '[]');
```
Comment (geekyd) [inline app/components/ui-table-server.js]: Great work!

**Comment by harshitagupta30** [comment]
@CosmicCoder96 @niranjan94 @geekyd Whenever I am trying to access the deployment link on admin/reports route is showing internal error 500. I am unable to identify rhe reason. Can you please tell?

**Pull request merged**